### DISCUSSION TITLE
Window associations not working

### DISCUSSION THREAD

**Discussion by khaalidrafi** [keepassxreboot/keepassxc#6297] [Q&A]
Hi. I'm using keepassxc on Arch Linux. When I set a window association for an entry and try to auto type on that associated window title, KeepassXC says there is no such entry. Why?

**Comment by droidmonkey** [comment]
Could be a million reasons. Can you please be more specific? Have you tried adding wildcards (*) to the name to prevent dynamic values from messing up the search? Are you picking the window title from the list of available ones or just typing it in yourself?
